shared/models/activitypub/objects/common-objects.ts

```diff
@@ -81,6 +81,12 @@ export interface ActivityMentionObject {
   name: string
 }
 
+export interface ActivityFlagReasonObject {
+  type: 'Hashtag' | 'Mention'
```

Comment:
You don't use `Mention` (please also fix `ActivityHashTagObject` and `ActivityMentionObject` `type`)